integration/combination/test_state_machine_with_cwe_dlq_generated.py

```diff
@@ -110,6 +110,6 @@ def test_state_machine_with_cwe(self):
 
 
 def get_first_key_value_pair_in_dict(dictionary):
-    key = list(dictionary.keys())[0]
+    key = next(iter(dictionary.keys()))
```

Comment:
One risk of this is we have special handling of `IndexError` and it will break if it becomes `StopIteration` error. I searched the codebase and didn't find any, so it should be safe